google_compute_engine/accounts/accounts_utils.py

```diff
@@ -46,10 +46,11 @@ def __init__(self, logger, groups=None, remove=False):
     self.google_sudoers_file = '/etc/sudoers.d/google_sudoers'
     self.google_users_dir = '/var/lib/google'
     self.google_users_file = os.path.join(self.google_users_dir, 'google_users')
+    self.default_groups = [ self.google_sudoers_group, 'docker', ]
```

Comment:
Adding a default group belongs as part of the instance config defaults file:
https://github.com/GoogleCloudPlatform/compute-image-packages/blob/master/google_compute_engine/instance_setup/instance_config.py#L44